packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -130,7 +130,6 @@ const baseConfig = (target) => {
                     extensions: ['.ts', '.tsx', '.js', '.jsx', '.json'],
                     alias: {
                         'babel-runtime': findInProjectThenSDK('babel-runtime'),
-                        '@tanstack/react-query': findInProjectThenSDK('@tanstack/react-query'),
```

Comment:
FYI this is the difference that this PR introduces. Compare to it to the original PR #1492 (see this [comment](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1492/files#r1369021505) in particular)